### ISSUE TITLE
VBR-encoded MP3s desync when starting playback mid-chart

### ISSUE THREAD

**Issue by NickHam13** [RHeavenStudio/HeavenStudio#490]
### Describe the bug

Title really says it all. It's mentioned in a different issue, but as this is a major problem and that issue is kinda all over the place I think it's a good idea to give it it's own issue so it can be properly tracked and closed when complete.

### How to reproduce the bug?

1. Use a playback position that is literally anywhere but the beginning

### Screenshots/Videos

https://github.com/RHeavenStudio/HeavenStudio/assets/49380508/59b4a8dd-83a2-4aec-9874-3d63b97bef9c

### What Operating System(s) were you using?

Windows 10/11

### Log file

```shell
Mono path[0] = 'C:/Users/PC/Documents/HeavenStudio/Heaven Studio_Data/Managed'
Mono config path = 'C:/Users/PC/Documents/HeavenStudio/MonoBleedingEdge/etc'
Initialize engine version: 2021.3.21f1 (1b156197d683)
[Subsystems] Discovering subsystems at path C:/Users/PC/Documents/HeavenStudio/Heaven Studio_Data/UnitySubsystems
GfxDevice: creating device client; threaded=1; jobified=1
Direct3D:
    Version:  Direct3D 11.0 [level 11.1]
    Renderer: NVIDIA GeForce GTX 1070 Ti (ID=0x1b82)
    Vendor:   NVIDIA
    VRAM:     8073 MB
    Driver:   31.0.15.3598
Begin MonoManager ReloadAssembly
- Completed reload, in  0.202 seconds
<RI> Initializing input.

<RI> Input initialized.

<RI> Initialized touch support.

UnloadTime: 0.990200 ms
Flushing possible JoyShocks...
Found 1 JoyShocks.
Connected 1 JoyShocks.
Setting up JoyShock: ( Handle 0, type 5 )
The character used for Underline is not available in font asset [rodin_lat_cy_ja_ko_spec SDF].
Unloading 6 Unused Serialized files (Serialized files now loaded: 0)
UnloadTime: 4.368100 ms
Running game loader RvlBadmintonLoader
Running game loader CtrBearLoader
Running game loader RvlBoardMeetingLoader
Running game loader NtrFlickLoader
Running game loader CtrCatchLoader
Running game loader RvlBookLoader
Running game loader AgbClapLoader
Running game loader NtrCoinLoader
Running game loader NtrCropLoader
Running game loader NtrDjLoader
Running game loader NtrDogNinjaLoader
Running game loader RvlDoubleDateLoader
Running game loader CtrDrummingLoader
Running game loader NtrIdolLoader
Running game loader AgbFireworkLoader
Running game loader CtrFirstContact
Running game loader RvlFlipperFlopLoader
Running game loader RvlForkLoader
Running game loader NtrGleeClubLoader
Running game loader RvlNewKarateLoader
Running game loader CtrTeppanLoader
Running game loader RvlRocketLoader
Running game loader NtrBackbeatLoader
Running game loader AgbMarcherLoader
Running game loader pcoMeatLoader
Running game loader AgbUpbeatLoader
Running game loader ntrMunchyMonkLoader
Running game loader NtrOctopusMachineLoader
Running game loader CtrPillowLoader
Running game loader AgbQuizShowLoader
Running game loader NtrPingpongLoader
Running game loader PcoSomenLoader
Running game loader AgbHairLoader
Running game loader RvlRingsideLoader
Running game loader NtrRockersLoader
Running game loader NtrSamuraiLoader
Running game loader RvlSeeSawLoader
Running game loader AgbGhostLoader
Running game loader AgbSpaceDanceLoader
Running game loader NtrSoccerLoader
Running game loader AgbBatterLoader
Running game loader RvlTambourineLoader
Running game loader AgbTapLoader
Running game loader RvlTapTroupeLoader
Running game loader NtrDazzlesLoader
Running game loader AgbTossBoysLoader
Running game loader AgbTramLoader
Running game loader MobTrickLoader
Running game loader NtrTunnelLoader
Running game loader AgbWaltzLoader
Running game loader RvlWorkingDoughLoader
Discord status updated
Create TimingDisplayComponent
Create SkillStarComponent
Create SectionComponent
Unloading 595 unused Assets to reduce memory usage. Loaded Objects now: 6984.
Total: 4.040500 ms (FindLiveObjects: 0.510500 ms CreateObjectMapping: 0.108100 ms MarkObjects: 3.122200 ms  DeleteObjects: 0.299400 ms)

Imported RIQ successfully!
Detected "v1" riq (RiqBeatmapData)
Running beatmap update handlers
Preprocessing beatmap...
Running entity update handlers
ASYNC loading assetbundles for game marchingOrders
ASYNC loading assetbundles for game marchingOrders
== Playthrough statistics of New Remix (played at 2023-06-24 00:02:41) ==
Average input offset for playthrough: 0ms
Accuracy for playthrough:  0.00
Cleared 0 sections out of 0
Skill Star not collected
ASYNC loading assetbundles for game marchingOrders
ASYNC loading assetbundles for game marchingOrders
HeavenStudio
Discord status updated
loaded song
Playing at 0
ASYNC loading assetbundles for game lockstep
Preloading game marchingOrders
Playing sound sequence zentai at beat 3
Preloading game lockstep
Playing sound sequence susume at beat 6
== Playthrough statistics of New Remix (played at 2023-06-24 00:03:35) ==
Average input offset for playthrough: 0ms
Accuracy for playthrough:  0.00
Cleared 0 sections out of 0
Skill Star not collected
Playing at 17.25
ASYNC loading assetbundles for game gleeClub
ASYNC loading assetbundles for game cropStomp
Pitch: 1.189207 Volume: 0.62 Name: rockers/strings/normal/1
Pitch: 1.122462 Volume: 0.62 Name: rockers/strings/normal/2
Pitch: 1 Volume: 0.62 Name: rockers/strings/normal/3
Pitch: 1.259921 Volume: 0.62 Name: rockers/strings/normal/4
Pitch: 1.587401 Volume: 0.62 Name: rockers/strings/normal/5
Preloading game gleeClub
Preloading game cropStomp
== Playthrough statistics of New Remix (played at 2023-06-24 00:03:45) ==
Average input offset for playthrough: 0ms
Accuracy for playthrough:  0.00
Cleared 0 sections out of 0
Skill Star not collected
Playing at 17.25
== Playthrough statistics of New Remix (played at 2023-06-24 00:03:56) ==
Average input offset for playthrough: 0ms
Accuracy for playthrough:  0.00
Cleared 0 sections out of 0
Skill Star not collected
Perfect Clear!
Playing at 0
Playing sound sequence zentai at beat 3
Playing sound sequence susume at beat 6
Pitch: 1.189207 Volume: 0.62 Name: rockers/strings/normal/1
Pitch: 1.122462 Volume: 0.62 Name: rockers/strings/normal/2
Pitch: 1 Volume: 0.62 Name: rockers/strings/normal/3
Pitch: 1.259921 Volume: 0.62 Name: rockers/strings/normal/4
Pitch: 1.587401 Volume: 0.62 Name: rockers/strings/normal/5
Pitch: 1.189207 Volume: 0.62 Name: rockers/strings/normal/1
Pitch: 1.122462 Volume: 0.62 Name: rockers/strings/normal/2
Pitch: 1 Volume: 0.62 Name: rockers/strings/normal/3
Pitch: 1.259921 Volume: 0.62 Name: rockers/strings/normal/4
Pitch: 1.587401 Volume: 0.62 Name: rockers/strings/normal/5
ASYNC loading assetbundles for game dogNinja
Preloading game dogNinja
== Playthrough statistics of New Remix (played at 2023-06-24 00:04:24) ==
Average input offset for playthrough: 0ms
Accuracy for playthrough:  88.57
Cleared 0 sections out of 0
Skill Star not collected
<color=#0099bc><b>DOTWEEN ► </b></color>Target or field is missing/null () ► 

  at (wrapper managed-to-native) UnityEngine.Transform.get_localScale_Injected(UnityEngine.Transform,UnityEngine.Vector3&)
  at UnityEngine.Transform.get_localScale () [0x00000] in <670c708f396c4df4bd34eacc99554521>:0 
  at DG.Tweening.ShortcutExtensions+<>c__DisplayClass46_0.<DOScaleX>b__0 () [0x00000] in <de1f8ef4975b4eacbeb0accc1b22c9b0>:0 
  at (wrapper delegate-invoke) DG.Tweening.Core.DOGetter`1[UnityEngine.Vector3].invoke_T()
  at DG.Tweening.Plugins.Vector3Plugin.EvaluateAndApply (DG.Tweening.Plugins.Options.VectorOptions options, DG.Tweening.Tween t, System.Boolean isRelative, DG.Tweening.Core.DOGetter`1[T] getter, DG.Tweening.Core.DOSetter`1[T] setter, System.Single elapsed, UnityEngine.Vector3 startValue, UnityEngine.Vector3 changeValue, System.Single duration, System.Boolean usingInversePosition, DG.Tweening.Core.Enums.UpdateNotice updateNotice) [0x000d9] in <de1f8ef4975b4eacbeb0accc1b22c9b0>:0 
  at DG.Tweening.Core.TweenerCore`3[T1,T2,TPlugOptions].ApplyTween (System.Single prevPosition, System.Int32 prevCompletedLoops, System.Int32 newCompletedSteps, System.Boolean useInversePosition, DG.Tweening.Core.Enums.UpdateMode updateMode, DG.Tweening.Core.Enums.UpdateNotice updateNotice) [0x00043] in <de1f8ef4975b4eacbeb0accc1b22c9b0>:0 

Disconnecting JoyShocks...
Clearing RIQ Cache...
Setting up 6 worker threads for Enlighten.
Releasing render texture that is set as Camera.targetTexture!
Releasing render texture that is set as Camera.targetTexture!
Releasing render texture that is set as Camera.targetTexture!
<color=#ff7337><color=#0099bc><b>DOTWEEN ► </b></color> SAFE MODE ►</color> DOTween's safe mode captured 1 errors. This is usually ok (it's what safe mode is there for) but if your game is encountering issues you should set Log Behaviour to Default in DOTween Utility Panel in order to get detailed warnings when an error is captured (consider that these errors are always on the user side).
- 1 missing target or field errors
Memory Statistics:
[ALLOC_TEMP_TLS] TLS Allocator
  StackAllocators : 
    [ALLOC_TEMP_MAIN]
      Peak usage frame count: [0-1.0 KB]: 6867 frames, [2.0 KB-4.0 KB]: 1 frames, [8.0 KB-16.0 KB]: 23 frames, [16.0 KB-32.0 KB]: 41681 frames, [32.0 KB-64.0 KB]: 10 frames, [64.0 KB-128.0 KB]: 55 frames, [128.0 KB-256.0 KB]: 4 frames, [256.0 KB-0.5 MB]: 1 frames, [0.5 MB-1.0 MB]: 1 frames, [2.0 MB-4.0 MB]: 1 frames
      Initial Block Size 4.0 MB
      Current Block Size 4.0 MB
      Peak Allocated Bytes 2.9 MB
      Overflow Count 0
    [ALLOC_TEMP_Loading.AsyncRead]
      Initial Block Size 64.0 KB
      Current Block Size 64.0 KB
      Peak Allocated Bytes 60.7 KB
      Overflow Count 0
    [ALLOC_TEMP_Loading.PreloadManager]
      Initial Block Size 256.0 KB
      Current Block Size 492.0 KB
      Peak Allocated Bytes 444.7 KB
      Overflow Count 17
    [ALLOC_TEMP_Background Job.Worker 8]
      Initial Block Size 32.0 KB
      Current Block Size 32.0 KB
      Peak Allocated Bytes 54 B
      Overflow Count 0
    [ALLOC_TEMP_Job.Worker 6]
      Initial Block Size 256.0 KB
      Current Block Size 256.0 KB
      Peak Allocated Bytes 36.1 KB
      Overflow Count 0
    [ALLOC_TEMP_Job.Worker 0]
      Initial Block Size 256.0 KB
      Current Block Size 256.0 KB
      Peak Allocated Bytes 36.1 KB
      Overflow Count 0
    [ALLOC_TEMP_Background Job.Worker 10]
      Initial Block Size 32.0 KB
      Current Block Size 32.0 KB
      Peak Allocated Bytes 54 B
      Overflow Count 0
    [ALLOC_TEMP_Background Job.Worker 9]
      Initial Block Size 32.0 KB
      Current Block Size 32.0 KB
      Peak Allocated Bytes 3.4 KB
      Overflow Count 1
    [ALLOC_TEMP_Job.Worker 5]
      Initial Block Size 256.0 KB
      Current Block Size 256.0 KB
      Peak Allocated Bytes 36.1 KB
      Overflow Count 0
    [ALLOC_TEMP_Background Job.Worker 14]
      Initial Block Size 32.0 KB
      Current Block Size 32.0 KB
      Peak Allocated Bytes 54 B
      Overflow Count 0
    [ALLOC_TEMP_Background Job.Worker 6]
      Initial Block Size 32.0 KB
      Current Block Size 32.0 KB
      Peak Allocated Bytes 22.2 KB
      Overflow Count 0
    [ALLOC_TEMP_Job.Worker 4]
      Initial Block Size 256.0 KB
      Current Block Size 256.0 KB
      Peak Allocated Bytes 36.1 KB
      Overflow Count 0
    [ALLOC_TEMP_ReadCacheIndex]
      Initial Block Size 64.0 KB
      Current Block Size 64.0 KB
      Peak Allocated Bytes 0.5 KB
      Overflow Count 0
    [ALLOC_TEMP_Job.Worker 3]
      Initial Block Size 256.0 KB
      Current Block Size 256.0 KB
      Peak Allocated Bytes 35.1 KB
      Overflow Count 0
    [ALLOC_TEMP_Background Job.Worker 12]
      Initial Block Size 32.0 KB
      Current Block Size 32.0 KB
      Peak Allocated Bytes 54 B
      Overflow Count 0
    [ALLOC_TEMP_EnlightenWorker] x 6
      Initial Block Size 64.0 KB
      Current Block Size 64.0 KB
      Peak Allocated Bytes 54 B
      Overflow Count 0
    [ALLOC_TEMP_Background Job.Worker 15]
      Initial Block Size 32.0 KB
      Current Block Size 32.0 KB
      Peak Allocated Bytes 54 B
      Overflow Count 0
    [ALLOC_TEMP_Background Job.Worker 1]
      Initial Block Size 32.0 KB
      Current Block Size 32.0 KB
      Peak Allocated Bytes 231 B
      Overflow Count 0
    [ALLOC_TEMP_Background Job.Worker 2]
      Initial Block Size 32.0 KB
      Current Block Size 32.0 KB
      Peak Allocated Bytes 6.5 KB
      Overflow Count 1
    [ALLOC_TEMP_Background Job.Worker 7]
      Initial Block Size 32.0 KB
      Current Block Size 32.0 KB
      Peak Allocated Bytes 54 B
      Overflow Count 0
    [ALLOC_TEMP_UnityGfxDeviceWorker]
      Initial Block Size 256.0 KB
      Current Block Size 256.0 KB
      Peak Allocated Bytes 6.8 KB
      Overflow Count 0
    [ALLOC_TEMP_AssetGarbageCollectorHelper] x 11
      Initial Block Size 64.0 KB
      Current Block Size 64.0 KB
      Peak Allocated Bytes 0 B
      Overflow Count 0
    [ALLOC_TEMP_Background Job.Worker 5]
      Initial Block Size 32.0 KB
      Current Block Size 32.0 KB
      Peak Allocated Bytes 1.6 KB
      Overflow Count 1
    [ALLOC_TEMP_Job.Worker 9]
      Initial Block Size 256.0 KB
      Current Block Size 256.0 KB
      Peak Allocated Bytes 35.1 KB
      Overflow Count 0
    [ALLOC_TEMP_Background Job.Worker 13]
      Initial Block Size 32.0 KB
      Current Block Size 32.0 KB
      Peak Allocated Bytes 54 B
      Overflow Count 0
    [ALLOC_TEMP_Job.Worker 8]
      Initial Block Size 256.0 KB
      Current Block Size 256.0 KB
      Peak Allocated Bytes 35.1 KB
      Overflow Count 0
    [ALLOC_TEMP_Job.Worker 7]
      Initial Block Size 256.0 KB
      Current Block Size 256.0 KB
      Peak Allocated Bytes 36.1 KB
      Overflow Count 0
    [ALLOC_TEMP_Job.Worker 1]
      Initial Block Size 256.0 KB
      Current Block Size 256.0 KB
      Peak Allocated Bytes 35.1 KB
      Overflow Count 0
    [ALLOC_TEMP_Job.Worker 2]
      Initial Block Size 256.0 KB
      Current Block Size 256.0 KB
      Peak Allocated Bytes 35.1 KB
      Overflow Count 0
    [ALLOC_TEMP_Background Job.Worker 3]
      Initial Block Size 32.0 KB
      Current Block Size 32.0 KB
      Peak Allocated Bytes 54 B
      Overflow Count 0
    [ALLOC_TEMP_Background Job.Worker 11]
      Initial Block Size 32.0 KB
      Current Block Size 32.0 KB
      Peak Allocated Bytes 3.0 KB
      Overflow Count 1
    [ALLOC_TEMP_Job.Worker 10]
      Initial Block Size 256.0 KB
      Current Block Size 256.0 KB
      Peak Allocated Bytes 36.1 KB
      Overflow Count 0
    [ALLOC_TEMP_Background Job.Worker 0]
      Initial Block Size 32.0 KB
      Current Block Size 32.0 KB
      Peak Allocated Bytes 54 B
      Overflow Count 0
    [ALLOC_TEMP_Background Job.Worker 4]
      Initial Block Size 32.0 KB
      Current Block Size 32.0 KB
      Peak Allocated Bytes 319 B
      Overflow Count 1
    [ALLOC_TEMP_BatchDeleteObjects]
      Initial Block Size 64.0 KB
      Current Block Size 64.0 KB
      Peak Allocated Bytes 54 B
      Overflow Count 0
[ALLOC_DEFAULT] Dual Thread Allocator
  Peak main deferred allocation count 1428
    [ALLOC_BUCKET]
      Large Block size 4.0 MB
      Used Block count 1
      Peak Allocated bytes 3.9 MB
      Failed Allocations. Bucket layout:
        16B: 29 Subsections = 29696 buckets. Failed count: 0
        32B: 21 Subsections = 10752 buckets. Failed count: 0
        48B: 50 Subsections = 17066 buckets. Failed count: 137
        64B: 38 Subsections = 9728 buckets. Failed count: 48
        80B: 63 Subsections = 12902 buckets. Failed count: 0
        96B: 18 Subsections = 3072 buckets. Failed count: 0
        112B: 14 Subsections = 2048 buckets. Failed count: 0
        128B: 23 Subsections = 2944 buckets. Failed count: 11
    [ALLOC_DEFAULT_MAIN]
      Peak usage frame count: [8.0 MB-16.0 MB]: 6893 frames, [16.0 MB-32.0 MB]: 41747 frames, [32.0 MB-64.0 MB]: 4 frames
      Requested Block Size 16.0 MB
      Peak Block count 2
      Peak Allocated memory 41.0 MB
      Peak Large allocation bytes 16.0 MB
    [ALLOC_DEFAULT_THREAD]
      Peak usage frame count: [16.0 MB-32.0 MB]: 48639 frames, [32.0 MB-64.0 MB]: 5 frames
      Requested Block Size 16.0 MB
      Peak Block count 1
      Peak Allocated memory 63.2 MB
      Peak Large allocation bytes 56.3 MB
[ALLOC_TEMP_JOB_1_FRAME]
  Initial Block Size 2.0 MB
  Used Block Count 1
  Overflow Count (too large) 0
  Overflow Count (full) 0
[ALLOC_TEMP_JOB_2_FRAMES]
  Initial Block Size 2.0 MB
  Used Block Count 1
  Overflow Count (too large) 0
  Overflow Count (full) 0
[ALLOC_TEMP_JOB_4_FRAMES (JobTemp)]
  Initial Block Size 2.0 MB
  Used Block Count 2
  Overflow Count (too large) 0
  Overflow Count (full) 0
[ALLOC_TEMP_JOB_ASYNC (Background)]
  Initial Block Size 1.0 MB
  Used Block Count 3
  Overflow Count (too large) 0
  Overflow Count (full) 0
[ALLOC_GFX] Dual Thread Allocator
  Peak main deferred allocation count 2
    [ALLOC_BUCKET]
      Large Block size 4.0 MB
      Used Block count 1
      Peak Allocated bytes 3.9 MB
      Failed Allocations. Bucket layout:
        16B: 29 Subsections = 29696 buckets. Failed count: 0
        32B: 21 Subsections = 10752 buckets. Failed count: 0
        48B: 50 Subsections = 17066 buckets. Failed count: 137
        64B: 38 Subsections = 9728 buckets. Failed count: 48
        80B: 63 Subsections = 12902 buckets. Failed count: 0
        96B: 18 Subsections = 3072 buckets. Failed count: 0
        112B: 14 Subsections = 2048 buckets. Failed count: 0
        128B: 23 Subsections = 2944 buckets. Failed count: 11
    [ALLOC_GFX_MAIN]
      Peak usage frame count: [1.0 MB-2.0 MB]: 6892 frames, [2.0 MB-4.0 MB]: 1 frames, [4.0 MB-8.0 MB]: 41751 frames
      Requested Block Size 16.0 MB
      Peak Block count 1
      Peak Allocated memory 5.6 MB
      Peak Large allocation bytes 0 B
    [ALLOC_GFX_THREAD]
      Peak usage frame count: [128.0 KB-256.0 KB]: 6893 frames, [0.5 MB-1.0 MB]: 41751 frames
      Requested Block Size 16.0 MB
      Peak Block count 1
      Peak Allocated memory 0.8 MB
      Peak Large allocation bytes 0 B
[ALLOC_CACHEOBJECTS] Dual Thread Allocator
  Peak main deferred allocation count 751
    [ALLOC_BUCKET]
      Large Block size 4.0 MB
      Used Block count 1
      Peak Allocated bytes 3.9 MB
      Failed Allocations. Bucket layout:
        16B: 29 Subsections = 29696 buckets. Failed count: 0
        32B: 21 Subsections = 10752 buckets. Failed count: 0
        48B: 50 Subsections = 17066 buckets. Failed count: 137
        64B: 38 Subsections = 9728 buckets. Failed count: 48
        80B: 63 Subsections = 12902 buckets. Failed count: 0
        96B: 18 Subsections = 3072 buckets. Failed count: 0
        112B: 14 Subsections = 2048 buckets. Failed count: 0
        128B: 23 Subsections = 2944 buckets. Failed count: 11
    [ALLOC_CACHEOBJECTS_MAIN]
      Peak usage frame count: [8.0 MB-16.0 MB]: 7635 frames, [16.0 MB-32.0 MB]: 41009 frames
      Requested Block Size 4.0 MB
      Peak Block count 3
      Peak Allocated memory 20.1 MB
      Peak Large allocation bytes 9.7 MB
    [ALLOC_CACHEOBJECTS_THREAD]
      Peak usage frame count: [1.0 MB-2.0 MB]: 7591 frames, [2.0 MB-4.0 MB]: 8980 frames, [4.0 MB-8.0 MB]: 2186 frames, [8.0 MB-16.0 MB]: 29887 frames
      Requested Block Size 4.0 MB
      Peak Block count 4
      Peak Allocated memory 14.9 MB
      Peak Large allocation bytes 0 B
[ALLOC_TYPETREE] Dual Thread Allocator
  Peak main deferred allocation count 0
    [ALLOC_BUCKET]
      Large Block size 4.0 MB
      Used Block count 1
      Peak Allocated bytes 3.9 MB
      Failed Allocations. Bucket layout:
        16B: 29 Subsections = 29696 buckets. Failed count: 0
        32B: 21 Subsections = 10752 buckets. Failed count: 0
        48B: 50 Subsections = 17066 buckets. Failed count: 137
        64B: 38 Subsections = 9728 buckets. Failed count: 48
        80B: 63 Subsections = 12902 buckets. Failed count: 0
        96B: 18 Subsections = 3072 buckets. Failed count: 0
        112B: 14 Subsections = 2048 buckets. Failed count: 0
        128B: 23 Subsections = 2944 buckets. Failed count: 11
    [ALLOC_TYPETREE_MAIN]
      Peak usage frame count: [16.0 KB-32.0 KB]: 6893 frames, [32.0 KB-64.0 KB]: 742 frames, [256.0 KB-0.5 MB]: 12089 frames, [0.5 MB-1.0 MB]: 28920 frames
      Requested Block Size 2.0 MB
      Peak Block count 1
      Peak Allocated memory 0.6 MB
      Peak Large allocation bytes 0 B
    [ALLOC_TYPETREE_THREAD]
      Peak usage frame count: [4.0 KB-8.0 KB]: 6891 frames, [8.0 KB-16.0 KB]: 9711 frames, [128.0 KB-256.0 KB]: 2551 frames, [256.0 KB-0.5 MB]: 7184 frames, [0.5 MB-1.0 MB]: 22307 frames
      Requested Block Size 2.0 MB
      Peak Block count 1
      Peak Allocated memory 0.6 MB
      Peak Large allocation bytes 0 B
```

**Comment by minenice55** [comment]
ID'ed issue: positive offsets aren't applied when playback starts mid-song

**Comment by minenice55** [comment]
turns out this is an issue with attempting to seek in variable-bitrate mp3s. need to research.
![image](https://github.com/RHeavenStudio/HeavenStudio/assets/43734252/5409684b-6397-4468-aef3-b78ebc1d1c98)

**Comment by NickHam13** [comment]
Yeah, that tracks. The MP3 I'm using is one I compressed myself at v0 instead of CBR. Is it a temporary solution to just re-encode it at 320kbps?

**Comment by minenice55** [comment]
seems so, and it appears that seeking in VBR mp3s is a whole issue in and of itself. regardless the issue may not be with HS itself

**Comment by minenice55** [comment]
will be closed by #454 with audio conversion methods
